anime/components/awardspage/ActualVoting.vue

```diff
@@ -670,106 +659,145 @@ export default ({
 .anime-title {
   font-size: var(--h3);
   margin: 1rem 2rem;
+  height: fit-content;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+	-webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-box-reflect

Reply:
this is used at so many parts of the site and the alternative solution isn't as efficient... :(